docs/python/environments.md

```diff
@@ -8,33 +8,33 @@ DateApproved: 1/20/2023
 MetaDescription: Configuring Python Environments in Visual Studio Code
 MetaSocialImage: images/tutorial/social.png
 ---
-# Using Python environments in VS Code
+# Python environments in VS Code
 
-This article discusses the helpful Python environments features available in Visual Studio Code. An "environment" in Python is the context in which a Python program runs and consists of an interpreter and any number of installed packages.
+An "environment" in Python is the context in which a Python program runs that consists of an interpreter and any number of installed packages.
 
 > **Note**: If you'd like to become more familiar with the Python programming language, review [More Python resources](#more-python-resources).
 
-## Python environments
+## Types of Python environments
 
 ### Global environments
 
 By default, any Python interpreter installed runs in its own **global environment**. For example, if you just run `python`, `python3`, or `py` at a new terminal (depending on how you installed Python), you're running in that interpreter's global environment. Any packages that you install or uninstall affect the global environment and all programs that you run within it.
 
-Do note that if you install packages into your global environment, though, in time it will become crowded with potentially unrelated or unexpected packages and make it difficult to properly test an application. You typically want to create an environment for each workspace.
+> **Tip**: In Python, it is best practice to create an environment for each workspace.
```

Comment:
maybe "to create a virtual environment" and then link to the virtual environment section below? If someone stops reading at this sentence it could sound like it's a best practice to create a "global" env for each workspace.

Reply:
ugh but not really cause if people use conda envs or other types of venvs that are not local that is fine too.  So maybe something like:
"it is best practice to create a separate environment from each workspace, for example by using a local environment"